application/forms/IcingaDependencyForm.php

```diff
@@ -303,6 +303,19 @@ protected function addObjectsElement()
             }
         }
 
+        $this->addElement('text', 'redundancy_group', [
+            'label' => $this->translate('Redundancy Group'),
+            'description' => $this->translate(
+                'The dependency redundancy group. A name for a group of which'
+                . ' at least one single dependency must be fulfilled for the'
+                . ' whole dependency to be fulfilled.'
+            ),
+            'class' => "director-suggest",
+            'data-suggestion-context' => 'dependencyredundancygroups',
```

Comment:
```suggestion
            'data-suggestion-context' => 'dependencyRedundancyGroups',
```

I know that the methods are case insensitive in PHP, but it is still desirable to match cases for consistency and readability reasons.